apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -340,6 +340,29 @@
       },
       "expectedAlertValue": "updateMicState called with micState: isMicMuted:true, reason:1",
       "expectedTestAppValue": "updateMicState called with micState: isMicMuted:true"
+    },
+    {
+      "title": "joinMeeting API Call - Success",
+      "type": "callResponse",
+      "version": ">2.20.0",
+      "boxSelector": "#box_joinMeeting",
+      "inputValue": {
+        "joinWebUrl": "https://www.example.com/",
+        "source": "Other"
+      },
+      "expectedAlertValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\"",
+      "expectedTestAppValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\""
```

Comment:
actual input: `joinMeeting called with joinWebUrl: https://www.example.com/`, missing double quotes for URL.

Reply:
Assert failure from Cypress:

Timed out retrying after 30000ms: expected '<span#text_joinMeeting>' to have text 'joinMeeting called with joinWebUrl: "https://www.example.com/"', but the text was 'joinMeeting called with joinWebUrl: https://www.example.com/'

Reply:
My hypothesis is that, the differences are on `expectedTestAppvalue`. It means that maybe somewhere in your test app works improperly when displaying strings.

Reply:
Oh, okay! made the changes accordingly. Thanks!